addons/web/static/src/js/views/kanban/kanban_record_quick_create.js

```diff
@@ -255,6 +264,11 @@ var RecordQuickCreate = Widget.extend({
      * @param {MouseEvent} ev
      */
     _onWindowClicked: function (ev) {
+        // ignore window click if ignoreWindowClick is true
```

Comment:
that comment isn't really helpful. Either expain why you would want to ignore clicks, or remove the comment